docs/CONTRIBUTING-it.md

```diff
@@ -180,7 +180,15 @@ I nostri elenchi forniscono un set minimo di metadati: titoli, URLs, autori, pia
 ##### Autori
 
 - Vogliamo dare i crediti agli autori ove appropriato, anche ai traduttori!
-- Per i lavori tradotti, l'autore originale dovrebbe essere incluso.
+- Per i lavori tradotti, l'autore originale dovrebbe essere incluso. We recommend using [MARC relators](https://loc.gov/marc/relators/relaterm.html) to credit creators other than authors, as in this example:
+
+    ```markdown
+    * [A Translated Book](http://example.com/book-it.html) - John Doe, `trl.:` Mike The Translator
+    ```
+
+    here, the annotation `trl.:` uses the MARC relator code for "translator".
+- Use a comma `,` to delimit each item in the author list.
+- You can shorten author lists with "`et al.`".
```

Comment:
```suggestion
- Per i lavori tradotti, l'autore originale dovrebbe essere incluso. Consigliamo di usare [MARC relators](https://loc.gov/marc/relators/relaterm.html) per accreditare autori diversi dal creatore, per esempio:

    ```markdown
    * [Un libro tradotto](http://example.com/book-it.html) - John Doe, `trl.:` Mike The Translator
    ```

    qui, l'annotazione `trl.:` usa il codice MARC relativo ai "traduttori".
- Usa l virgola `,` per separare ogni elemento nella lista degli autori
- Puoi accorciare la lista degli autori scrivendo "`et al.`" (e altri).
```